### ISSUE TITLE
More control over mice methods 

### ISSUE THREAD

**Issue by ppanko** [PcAux-Package/PcAux#36]
Might be nice to give users a more modular interface for assigning mice's imputation methods. For instance, although `polr` is typically preferred for "normal" ordinal variable imputation, some of the more skewed variables may benefit from the use of `pmm` (van Buuren said it, not me). If the variable set given to PcAux contains both types of ordinal variables, the user may want to use different methods for different variables within the same "ordVars" set.

**Comment by AlmutZieher** [comment]
I agree. First off I want to say I'm not a programmer and my gaps in understanding the code are probably huge. Nonetheless, I have been working on imputation with two and three level models. I'd like to use PCAux (it makes sense to me to use this approach to imputation) but I have a better sense of what I'm doing with mice. I can designate my predictor matrix and imputation method matrix. It may  be that I could do more of this with PCAux using the control argument, but there are few examples so nothing to go by. There is also no way I've identified to impute 3 level data in PCAux. In mice I've used the miceadds package to try to do so; though I'm still struggling to get to code that actually runs.
